bin/kea/conf.rpi_3_32b_sjg-rpi_3b

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2016 Konsulko Group. All rights reserved.
```

Comment:
I hope we don't run into any hostname collisions:-) Perhaps I should have added an "owner" field in the path, or used full hostnames including the domain rather than just the hostname. Still, this is fine.

Reply:
I hope this becomes popular enough that this is an issue!